docs/pages/router/create-pages.mdx

```diff
@@ -112,3 +112,38 @@ export default function Page() {
   return <Text>Blog post: {slug}</Text>;
 }
 ```
+
+## Non-route files
+
+Every file and sub-directory inside the **app** directory is either a `_layout` file, or a route in your app. This means that other files like your components, hooks, utilities, and so on, cannot be placed in the **app** directory because they are neither screens nor layout files.
```

Comment:
```suggestion
Every file and sub-directory inside the **app** directory is either a `_layout` file or a route in your app. Other files, such as components, hooks, utilities, and so on, cannot be placed in the **app** directory because they are neither screens nor layout files.
```

There's no need to add a comma in the first line.
Let's remove the `"This means that"` since it doesn't add any benefit to the ongoing context.